### PR TITLE
Add sample for IndicateMissingValues

### PR DESCRIPTION
**Pull request by yaeldekel**

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/IndicateMissingValues.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class IndicateMissingValues
+    {
+        public static void Example()
```
Comment (sfilipi) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/IndicateMissingValues.cs]: >Example [](start = 27, length = 7)

please link it from the extension it documents. 

see the:
[https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs#L83](https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs#L83)

something like:

 /// <example>
        /// <format type="text/markdown">
        /// <CDATA[
        ///  [!code-csharp[KeyToValueMappingEstimator]
        /// ]]></format>
        /// </example>

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/IndicateMissingValues.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            // Create a TextLoader for the breast-cancer dataset, and load it into an IDataView.
+            var loader = mlContext.Data.CreateTextLoader(new[]
+            {
+                new TextLoader.Column("Label", DataKind.Single, 0),
+                new TextLoader.Column("Features1", DataKind.Single, 1, 5),
+                new TextLoader.Column("Features2", DataKind.Single, 7, 9),
+                new TextLoader.Column("Missing", DataKind.Single, 6)
+            });
+            var file = SamplesUtils.DatasetUtils.DownloadBreastCancerDataset();
+            var data = loader.Load(new MultiFileSource(file));
```
Comment (sfilipi) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/IndicateMissingValues.cs]: i think we want to move forward with just a small in-memory IEnumerable for the data, and skip the TextLoader and the real datasets.

Comment (shmoradims) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/IndicateMissingValues.cs]: you can find a sample with small in-memory dataset in this comments: [https://github.com/dotnet/machinelearning/issues/2726#issuecomment-468781807](https://github.com/dotnet/machinelearning/issues/2726#issuecomment-468781807)

---
In reply to: [261707265](https://github.com/dotnet/machinelearning/pull/2814#discussion_r261707265) [](ancestors = 261707265)

**Comment by sfilipi** [comment]
work got moved to here: https://github.com/dotnet/machinelearning/pull/2960

**Pull request closed without merging**